widgets/Spanner-batchsink.json

```diff
@@ -49,7 +49,7 @@
         },
         {
           "widget-type": "csv",
-          "label": "Primary keys",
+          "label": "Primary key",
```

Comment:
Key.

For consistency, words in labels should be capitalized unless they are 'a', 'an', 'of', 'the'